spec/rails_helper.rb

```diff
@@ -58,6 +48,12 @@
 
   config.before(:suite) do
     Rails.application.load_seed
+
+    begin
```

Comment:
@solipet Does this still accomplish what you wanted?   I'm not totally sure yet if doing a `.clear` will mess up the test suite in any way.

Reply:
Seems like `.info` would still work here?

Reply:
I thought so too but `undefined method `info' for #<Readthis::Cache:0x0000556fc61c2cb8>`

Reply:
oooo it's not actually a `Redis` .... this works:

```ruby
REDIS_POOL.with { |cache| cache.pool.with { |redis| redis.info } }
```

Reply:
OOOHHHhhhhh Nice. Thank you.